### PR TITLE
[Improve] Use isEmpty() instead of length() == 0 or size() == 0 for collections

### LINKED ISSUE #3334: [Improve] Use isEmpty() instead of length() == 0 or size() == 0 for collections
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-streampark/streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### Describe the proposal

This rule encourages developers to adopt a more semantic and cleaner way of checking for collections. It's a small but meaningful step towards improving the overall quality of our codebase.

### Task list

- [ ] org.apache.streampark.flink.connector.doris.util.DorisDelimiterParser#parse
- [ ] org.apache.streampark.console.core.service.impl.SqlCompleteServiceImpl.SqlCompleteFstTree#getDFSWord

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by caicancai**
[Improve] Use isEmpty() instead of length() == 0 or size() == 0 for collections

## What changes were proposed in this pull request

Issue Number: close #3334 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

**Comment by caicancai** [comment]
cc @wolfboys @RocMarshal

File: streampark-flink/streampark-flink-connector/streampark-flink-connector-doris/src/main/java/org/apache/streampark/flink/connector/doris/util/DorisDelimiterParser.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public class DorisDelimiterParser {
   private static final String HEX_STRING = "0123456789ABCDEF";
 
   public static String parse(String sp) throws RuntimeException {
-    if (sp == null || sp.length() == 0) {
+    if (sp == null || sp.isEmpty()) {
```
Comment (wolfboys) [inline streampark-flink/streampark-flink-connector/streampark-flink-connector-doris/src/main/java/org/apache/streampark/flink/connector/doris/util/DorisDelimiterParser.java]: StringUtils.isBlank()....

Comment (caicancai) [reply streampark-flink/streampark-flink-connector/streampark-flink-connector-doris/src/main/java/org/apache/streampark/flink/connector/doris/util/DorisDelimiterParser.java:27]: There seems to be no isblank in my idea completion

**Comment by caicancai** [comment]
cc @wolfboys @RocMarshal

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**